src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.cpp

```diff
@@ -333,16 +333,11 @@ Value foldOrEmitONNXSqueezeV11Op(ConversionPatternRewriter &rewriter,
   MultiDialectBuilder<KrnlBuilder, MathBuilder, MemRefBuilder, OnnxBuilder>
       create(rewriter, loc);
   if (krnl::isKrnlGlobalConstant(input) || isDenseONNXConstant(input)) {
-    char *inputBuffer = createArrayFromDenseElementsAttr(
-        input.getDefiningOp()
-            ->getAttrOfType<::mlir::Attribute>("value")
-            .dyn_cast_or_null<mlir::DenseElementsAttr>());
-    char *outputBuffer = allocateBufferFor(resultType, /*useMaxSize=*/false);
-    convertDoubleInt64ToExactType(resultType, inputBuffer, outputBuffer);
-    Value constVal =
-        create.onnx.constantFromRawBuffer(resultType, outputBuffer);
-    free(outputBuffer);
-    free(inputBuffer);
+    DenseElementsAttr inputElements =
+        getDenseElementAttributeFromONNXValue(input);
```

Comment:
TODO: I need to update this so it also handles the krnl.global constant case

same in lines 357-358 below

Reply:
fixed